packages/shared/styles/components/SfGroupedProduct.scss

```diff
@@ -0,0 +1,93 @@
+@import "../variables";
+
+$grouped-product-max-width: 7.8125rem !default;
+$grouped-product-max-width-desktop: 5.125rem !default;
+$grouped-product-font-family: $body-font-family-secondary !default;
+$grouped-product-font-size: $font-size-regular-mobile !default;
+$grouped-product-font-size-desktop: $font-size-extra-small-desktop !default;
+$grouped-product-font-weight: $body-font-weight-secondary !default;
+$grouped-product-line-height: 1.6 !default;
+$grouped-product__select-quantity-width: 3.125rem !default;
```

Comment:
do we need those? Shouldn't be this a a general purpose QuantitySelect  component?

Reply:
@filrak definitely we need SfQuantitySelector but now we used SfInput and add styles for it. I add issue for that #453